Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
We want to be broader than this here, really. It also fails with any Ruby that isn't the system's, such as `rbenv` (Which is what I tested it locally with). If we wanted to go down this route we'd probably want to do something like:

```ruby
ENV.prepend_path "PATH", `ruby -e 'puts RbConfig::CONFIG["bindir"]'`
```
I'm a little confused why this works without extra logic if we chuck in a `depends_on :ruby => "1.8", :build` dependency to be honest.